### DISCUSSION TITLE
Linux + Steam is it possible ?

### DISCUSSION THREAD

**Discussion by Ciflire** [Xzandro/sw-exporter#532] [Q&A]
I was wondering if it was a possibility, the steam version runs under linux, but with wine/proton it could be containerized an inaccessible.
Also i'm using a wierd distro, so i wanted to gather infos from the linux community to try and setup something on my pc
If anyone knows how to do it on nixos i'd love to hear it too.
I generated the certificate and launched sw with
```
NIX_SSL_CERT_FILE="/home/ciflire/.desktop/Summoners War Exporter Files/cert/ca.pem" %command%
```
which on my distro should add the certificate to root user

**Comment by EytanMorg** [comment]
nope. easier to use [waydroid](https://github.com/EytanMorg/sw-exporter/wiki/3.-Setup-guides#arch-linux-with-waydroid-only-do-this-if-you-know-what-you-are-doing)
